packages/core/src/browser/preferences/preference-service.ts

```diff
@@ -66,34 +66,163 @@ export interface PreferenceChanges {
 }
 
 export const PreferenceService = Symbol('PreferenceService');
+/**
+ * Service to store and retrieve preference values.
```

Comment:
The description here might not be accurate, while it does allow clients to store and retrieve preference values, they are also able to listen to preference change events in addition to other utilities.

Reply:
That's true, the description is too simplistic.